Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
+            while buf = rd.gets
+              log.puts buf
+              print "."
```

Comment:
Note I don't mind changing the logic in `test-bot` to be travis-specific or move the timer there.